src/assets/sass/protocol/components/_card.scss

```diff
@@ -70,20 +70,26 @@
     .mzp-c-card-title {
         @include text-display-sm;
         display: inline;
-        border-bottom: 1px solid transparent;
+        border-bottom: 2px solid transparent;
     }
 
     .mzp-c-card-desc {
         margin: $margin-xs 0 0;
     }
 
+    .mzp-c-card-cta {
+        font-weight: bold;
+        margin: $margin-xs 0 0;
+    }
+
     .mzp-c-card-block-link {
         border-radius: 2px;
         display: block;
 
         &:link,
         &:visited {
             color: $color-black;
+            border-bottom: 2px solid transparent;
```

Comment:
This doesn't seem to be required (unless I'm missing something, which is also likely)